README.md

```diff
@@ -4,6 +4,14 @@
 * Add AKS clusters to kubeconfig via the GUI
 * Run AppLens detector diagnostics on your AKS clusters
 
+## Telemetry
+
+This extension collects telemetry data to help us build a better experience for building applications with Kubernetes and VS Code. We only collect the following data:
+
+* Which commands are executed, and whether they are executed against an Azure cluster.
```

Comment:
We don't record whether they are executed against an Azure cluster - just remove that clause.